pkg/icingadb/ha.go

```diff
@@ -286,6 +288,13 @@ func (h *HA) realize(ctx context.Context, s *icingaredisv1.IcingaStatus, t *type
 				return internal.CantPerformQuery(err, stmt)
 			}
 
+			if takeover {
+				stmt := h.db.Rebind("UPDATE icingadb_instance SET responsible = ? WHERE environment_id = ? AND id <> ?")
+				_, err := tx.ExecContext(ctx, stmt, "n", envId, h.instanceId)
+
+				return internal.CantPerformQuery(err, stmt)
```

Comment:
Too eager error handling ...

Reply:
Why... exactly?

Reply:
You're always returning here, even if there was no error?